packages/code-studio/.env

```
@@ -10,7 +10,7 @@ REACT_APP_PLUGIN_URL=/ide/plugins/
 REACT_APP_INTERNAL_PLUGINS=
 REACT_APP_ENABLE_LOG_PROXY=true
 REACT_APP_SUPPORT_LINK=https://github.com/deephaven/web-client-ui/
-REACT_APP_DOCS_LINK=https://deephaven.io/open-core/docs
+REACT_APP_DOCS_LINK=https://deephaven.io/core/docs
```

```suggestion
REACT_APP_DOCS_LINK=https://deephaven.io/core/docs/
```

Site has trailing slashes, might as well match rather than it have it add it on.